### ISSUE TITLE
PHP 8.1 using PHP 8.2 ?

### ISSUE THREAD

**Issue by MatusBoa** [lbausch/laravel-ci#13]
Hey, after releasing 8.2 on this docker image my CI pipeline broke, because apparently it is using php 8.2.

Here are shortened logs

```
Pulling docker image lbausch/laravel-ci:php81 ...
....
$ php -v
PHP 8.2.0 (cli) (built: Dec  8 2022 15:03:15) (NTS)
```

**Comment by MatusBoa** [comment]
After closer look at build, I tried to delete php-redis package and magically cli version is now 8.1.

**Comment by lbausch** [comment]
Thanks for reporting! I just fixed this in 3585b0ff7c1304414e05164e07162fd42e310e09
```bash
$ docker run --rm lbausch/laravel-ci:php81 php -v
PHP 8.1.13 (cli) (built: Nov 26 2022 14:27:02) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.13, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.13, Copyright (c), by Zend Technologies
    with Xdebug v3.2.0, Copyright (c) 2002-2022, by Derick Rethans
```
